notes

```diff
@@ -26,7 +26,7 @@ Exchange Funder (deps: OKEX, GALOY)
 -------------------------
 Trader (deps: OKEX)
 -> HedgingNeed
-=> moves money to/from trading account ( controling collateral)
+=> moves money to/from trading account ( controlling collateral)
```

Comment:
did you accidentally commit your notes? Might want to add this to your global gitignore.